dds/DCPS/transport/rtps_udp/RtpsUdpReceiveStrategy.h

```diff
@@ -73,6 +75,7 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpReceiveStrategy
                                       const ACE_SOCK_Dgram& socket,
                                       ACE_INET_Addr& remote_address,
                                       ICE::Endpoint* endpoint,
+                                      RtpsUdpTransport* tport,
```

Comment:
Is this needed?  Doesn't the Data Link already have it?

Reply:
The helper is a free function.